### PR TITLE
php/peachpie test updated and fixed

### PR DESCRIPTION
**Pull request by jakubmisek**
- fixed build of the test image
- tests updated to latest framework

### CODE REVIEW

File: frameworks/PHP/peachpie/Website/db.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  // Database connection (TODO: When it works, use PDO instead)
+  $link = mysql_pconnect('tfb-database', 'benchmarkdbuser', 'benchmarkdbpass');
+  mysql_select_db('hello_world', $link);
+
+  $id = mt_rand(1, 10000);
+  $result = mysql_query("SELECT randomNumber FROM World WHERE id=$id", $link);
+
+  // Store result in array.
+  echo json_encode( ['id' => $id, 'randomNumber' => mysql_result($result, 0, 0)] );
```
Comment (joanhey) [inline frameworks/PHP/peachpie/Website/db.php:19]: Must be read the id and the randomNumber.
https://github.com/TechEmpower/FrameworkBenchmarks/issues/5167#issuecomment-544679539

File: frameworks/PHP/peachpie/Website/query.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  // Create an array with the response string.
+  $arr = [];
+  
+  // For each query, store the result set values in the response array
+  while ($query_count--) {
+    $id = mt_rand(1, 10000);
+    $result = mysql_query("SELECT randomNumber FROM World WHERE id=$id", $link);
+
+    // Store result in array.
+    $arr[] = ['id' => $id, 'randomNumber' => mysql_result($result, 0, 0)];
```
Comment (joanhey) [inline frameworks/PHP/peachpie/Website/query.php:27]: The same here.
https://github.com/TechEmpower/FrameworkBenchmarks/issues/5167#issuecomment-544679539

File: frameworks/PHP/peachpie/Website/db.php (hunk 1 above)
Comment (jakubmisek) [reply frameworks/PHP/peachpie/Website/db.php]: thank you

File: frameworks/PHP/peachpie/Website/query.php [hunk 3]
```diff
@@ -21,10 +21,10 @@ function query() {
   // For each query, store the result set values in the response array
   while ($query_count--) {
     $id = mt_rand(1, 10000);
-    $result = mysql_query("SELECT randomNumber FROM World WHERE id=$id", $link);
+    $result = mysql_query("SELECT id,randomNumber FROM World WHERE id=$id", $link);
 
     // Store result in array.
-    $arr[] = ['id' => $id, 'randomNumber' => mysql_result($result, 0, 0)];
+    $arr[] = ['id' => $id, 'randomNumber' => mysql_result($result, 0, 1)];
```
Comment (joanhey) [inline frameworks/PHP/peachpie/Website/query.php:27]: The id must be read from the db too.

File: frameworks/PHP/peachpie/Website/db.php [hunk 4]
```diff
@@ -13,10 +13,10 @@ function db() {
   mysql_select_db('hello_world', $link);
 
   $id = mt_rand(1, 10000);
-  $result = mysql_query("SELECT randomNumber FROM World WHERE id=$id", $link);
+  $result = mysql_query("SELECT id,randomNumber FROM World WHERE id=$id", $link);
 
   // Store result in array.
-  echo json_encode( ['id' => $id, 'randomNumber' => mysql_result($result, 0, 0)] );
+  echo json_encode( ['id' => $id, 'randomNumber' => mysql_result($result, 0, 1)] );
```
Comment (joanhey) [inline frameworks/PHP/peachpie/Website/db.php:19]: The id must be read from the db too.

Comment (jakubmisek) [reply frameworks/PHP/peachpie/Website/db.php:19]: I see, will be fixed. fyi php/hhvm (https://github.com/TechEmpower/FrameworkBenchmarks/blob/d454e5062b4742ae6c3eab230d042adba5c52a9f/frameworks/PHP/hhvm/once.php.inc#L39) is wrong as well, although it is tagged as "broken"

Comment (joanhey) [reply frameworks/PHP/peachpie/Website/db.php:19]: Need to be changed too.
Thank you

Comment (joanhey) [reply frameworks/PHP/peachpie/Website/db.php:19]: When anybody try to repair the hhvm, will be enforced to change it too.

**Comment by nbrady-techempower** [comment]
Thanks for reviewing these @joanhey !

**Comment by joanhey** [comment]
The test is still failing,
Perhaps will be good to revert it.

**Comment by nbrady-techempower** [comment]
@joanhey See, still early! Totally thought I saw a green check mark here. Please @jakubmisek create a new PR when you have this working. Sorry about that!

**Pull request merged**